bundles/org.openhab.ui/web/src/components/connection-health-mixin.js

```diff
@@ -0,0 +1,94 @@
+import mixin from 'reload-mixin.js'
```

Comment:
Thanks - this could have waited, enjoy your concert! (who is it?)
I'll tentatively do the testing tomorrow. The builds still fail, though:

```
[INFO] ERROR in ./src/components/connection-health-mixin.js
[INFO] Module not found: Error: Can't resolve 'reload-mixin.js' in '/home/runner/work/openhab-webui/openhab-webui/bundles/org.openhab.ui/web/src/components'
[INFO]  @ ./src/components/connection-health-mixin.js 1:0-36 3:11-16
```


```suggestion
import mixin from './reload-mixin'
```
As an habit I dropped the .js extensions in imports, except in `assets` where is it required to build the component docs (as it is run by Node which requires them).